### PR TITLE
Use immutable default values

### PR DESCRIPTION
**Pull request by maratori**

### CODE REVIEW

File: lona/server.py [hunk 1]
```diff
@@ -447,9 +447,9 @@ async def _handle_http_request(self, http_request):
             return response
 
         # websocket requests ##################################################
-        if(http_request.method == 'GET' and
-           http_request.headers.get('upgrade', '').lower() == 'websocket'):
-
+        if (http_request.method == 'GET' and
+                http_request.headers.get('upgrade', '').lower() ==
+                'websocket'):
```
Comment (fscherf) [inline lona/server.py:452]: This seems to be only a style change. Can you put this in another commit?

File: lona/view.py [hunk 2]
```diff
@@ -37,9 +37,8 @@ def _add_view_to_objects(cls, view_object):
     def _remove_view_from_objects(cls, view_object):
         objects_key = cls._get_objects_key(view_object, is_class=False)
 
-        if(objects_key in cls._objects and
-           view_object in cls._objects[objects_key]):
-
+        if (objects_key in cls._objects and
+                view_object in cls._objects[objects_key]):
```
Comment (fscherf) [inline lona/view.py:41]: This is also only a style change

**Review by fscherf — COMMENTED** [review]
Besides some nitpicking great PR!

File: lona/server.py (hunk 1 above)
Comment (maratori) [reply lona/server.py]: Removed this change. It was done by my IDE, not intentionally.

File: lona/view.py (hunk 2 above)
Comment (maratori) [reply lona/view.py]: Removed this change. It was done by my IDE, not intentionally.

**Comment by fscherf** [comment]
Great! Thanks! That looked like some tedious work
